www/js/diary/LabelTab.tsx

```diff
@@ -336,8 +330,8 @@ const LabelTab = () => {
    * @returns Confirmed mode, which could be a vehicle identity as determined by Bluetooth scans,
    *  or the label option from a user-given 'MODE' label, or undefined if neither exists.
    */
-  const confirmedModeFor = (tlEntry: TimelineEntry) =>
-    timelineBleMap?.[tlEntry._id.$oid] || labelFor(tlEntry, 'MODE');
+  const confirmedModeFor = (tlEntry: CompositeTrip) =>
+    primarySectionForTrip(tlEntry)?.ble_sensed_mode || labelFor(tlEntry, 'MODE');
```

Comment:
nit, future fix: I think it would be worthwhile for you and Abby to sync up on how this should work across the public dashboard and the phone. The server side data does not have a `confirmedMode`; maybe it should add one. Also, I am not sure that `confirmed_mode` is the best name for this, since the ble_sensed_mode has not been confirmed by anybody.

Reply:
Agreed, this is not the long term solution.

"Confirmed mode" was something temporary I threw together for the alpha. It actually represents the BLE sensed mode OR the labeled mode, depending on what is available.

For `dfc-fermata` there are no mode labels and I think it makes more sense to use something like `primary_ble_sensed_mode`. will coordinate this with Abby once I know what adjustments we are making to the section summaries 

I do think something like "confirmed mode" will be useful in the app dashboard and for analysis. I'm envisioning future programs that could theoretically use BLE sensing in addition to mode+purpsoe labels – the BLE sensed trips will not need manual user input since we already "confirmed" the mode when we matched it to a beacon.
When we do viz/ analysis we will want one property that encompasses both cases: i) determined by BLE or ii) manually confirmed